index.html

```diff
@@ -2,24 +2,70 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="initial-scale=1, width=device-width,
-      shrink-to-fit=no">
-    <title>My Portfolio Website</title>
-    <link rel="stylesheet" href="./css/style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width" , initial-scale="1.0">
+    <title>My Portfolio website</title>
+
+    <!-- link external stylesheet -->
+    <link href="css/style.css" rel="stylesheet">
+
+    <!-- link fontawesome stylesheet -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+
+    <!-- link google fonts stylesheet -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Mulish:wght@200&display=swap"
+      rel="stylesheet">
+    <script src="https://kit.fontawesome.com/70cd6cd3b0.js"
+      crossorigin="anonymous"></script>
   </head>
 
   <body>
-    <!-- header section -->
-    <section class="header">
+    <!-- first section (header) -->
+    <section>
+      <header>
+        <div><a href="#">Mohamed Eldimardash</a></div>
+        <nav class="menu">
+          <a href="#"><img src="images/burger.png" alt="burger-menu"></a>
+        </nav>
+      </header>
+    </section> <!-- end section 1 -->
+
+    <!-- second section (headline) -->
+    <section>
+      <main class="headline">
+        <div>
+          <img src="images/bg.png" alt="burger-menu" class="bg-image">
+        </div>
+        <h1>Hey There. <br> I'm Mohamed</h1>
+        <h2>I am a Software Developer</h2>
+        <div>
+          <p>I can help build you a product, feature or website. Look through
+            some
+            of my work and experience! If you like what you see and have a
+            project
+            you need coded, don't hesitate to contact me.</p>
+        </div>
 
-    </section>
+        <!-- social link icons -->
+        <ul class="socials">
+          <li>
+            <a href="#"><i class="fa-brands fa-github"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-regular fa-hand-peace"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+        </ul>
```

Comment:
- [ ] Please kindly use the social media `images icon` in the Figma design, and also align the entire `ul` to the `left` and not to the `center` as shown in [the template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- [ ] Your design 👇 
![social](https://user-images.githubusercontent.com/78251927/205149161-78c086a2-a0bd-44a7-8e3f-5ddfab05ffd3.PNG)

- [ ] Figma design
![social2](https://user-images.githubusercontent.com/78251927/205149231-53cfbede-7c38-4ee8-8542-d2862c82f202.PNG)